sdk/core/azure-core/dev_requirements.txt

```diff
@@ -1,4 +1,5 @@
 trio
+flask==2.0.3
```

Comment:
```suggestion
```

Reply:
I did investigation. The presence of the pin here is why it fails during build. My fault.